hooks/addon

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -eu
+vault="secret/$GENESIS_VAULT_PREFIX"
+target=$GENESIS_ENVIRONMENT
+
+
+list() {
+  echo "The following addons are defined:"
+  echo
+  echo "  list                List out all supported addons."
+  echo "  visit-prometheus    Visit the Prometheus Web UI (macOS only)"
+  echo "  visit-grafana       Visit the Grafana dashboard (macOS only)"
+  echo "  visit-alertmanager  Visit the AlertManager dashboard (macOS only)"
+  echo "  runtime-config      Generate a runtime configuration for node-exporter"
+  echo
```

Comment:
Add the shortcuts.  See https://github.com/genesis-community/shield-genesis-kit/pull/12/files#diff-f2ed0d9452b9c10472dec6ebf93357afR13